### ISSUE TITLE
[GeoTiff] Upload with auxiliary file fails

### ISSUE THREAD

**Issue by ridoo** [GeoNode/geonode-importer#196]
Uploader does not accept a `tfw` auxiliary file along with a GeoTif anymore. According to https://github.com/GeoNode/geonode/issues/135 this was a working feature.

**Comment by giohappy** [comment]
@ridoo I think we missed this since several versions, at least from GeoNode 4 upwards.

Unfortunately, the configuration of the admitted extension is a bit broken at the moment. We have a configuration on the backend that is only partially reflected on the client because the client mixes its own logic for multi-file formats. This means that restoring the option to upload `tfw` requires some investigation both on the backend and the frontend.

**Comment by ridoo** [comment]
Good point .. thanks for the info @giohappy. Just wanted to start by adding it to the `supported_file_extension_config`, but when this will fall too short, I will postpone for now.
